packages/atlas/src/components/_inputs/Input/Input.stories.tsx

```diff
@@ -0,0 +1,124 @@
+import { useRef } from '@storybook/addons'
+import { Meta, Story } from '@storybook/react'
+import React, { useState } from 'react'
+
+import { Avatar } from '@/components/Avatar'
+import { Pill } from '@/components/Pill'
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { SvgActionBid, SvgActionClose, SvgActionSearch, SvgJoyTokenMonochrome16 } from '@/components/_icons'
+import { cVar } from '@/styles'
+import { formatNumber } from '@/utils/number'
+
+import { Input, InputProps } from '.'
+
+export default {
+  title: 'inputs/Input',
+  component: Input,
+  args: {
+    size: 'large',
+    type: 'text',
+    placeholder: 'placeholder text',
+  },
+  argTypes: {
+    value: { table: { disable: true } },
+    onChange: { table: { disable: true } },
+    onBlur: { table: { disable: true } },
+    onFocus: { table: { disable: true } },
+    onKeyDown: { table: { disable: true } },
+    onWheel: { table: { disable: true } },
+    autoComplete: { table: { disable: true } },
+    nodeEnd: { table: { disable: true } },
+    nodeStart: { table: { disable: true } },
+    className: { table: { disable: true } },
+    defaultValue: { table: { disable: true } },
+    actionButton: { table: { disable: true } },
+  },
+} as Meta
+
+const Template: Story<InputProps> = (args) => <Input {...args} />
+
+const TemplateWithControlledInput: Story<InputProps> = (args) => {
+  const [value, setValue] = useState('')
+  return <Input {...args} onChange={(e) => setValue(e.currentTarget.value)} value={value} />
+}
+
+const TemplateWithUncontrolledInput: Story<InputProps> = (args) => {
+  const ref = useRef<HTMLInputElement | null>(null)
+  return (
+    <>
+      <Input {...args} ref={ref} />
+      <div style={{ marginTop: 16 }}>
+        <Button onClick={() => alert(ref.current?.value)}>Show input value</Button>
+      </div>
+    </>
+  )
+}
+
+const TemplateWithPreffixAndSuffix: Story<InputProps> = (args) => {
+  const [dollars, setDollars] = useState<number>()
+  return (
+    <div style={{ display: 'grid', gap: 24 }}>
+      <Input
+        {...args}
+        nodeStart={<Pill label="label" />}
+        nodeEnd={
+          <Text secondary color={cVar('colorTextMuted')} variant="t300">
+            $
+          </Text>
```

Comment:
I'm not sure if this is captured in the Figma docs but when we discussed the suffix/prefix with Adam, he mentioned that it would be good for all those basic text nodes to always use the same styling.

What do you think about this: We could check whether `nodeStart` (or end) is a string and if it is, we could wrap it in this `<Text>` inside the `Input` itself, not requiring the consumers to repeat the style?

Btw, I think you don't need both `secondary` and `color` props

Reply:
good idea - added